spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/avro/AvroSchemaRegistryClientMessageConverter.java

```diff
@@ -132,19 +131,22 @@
 
 	private SubjectNamingStrategy subjectNamingStrategy;
 
+
 	/**
 	 * Creates a new instance, configuring it with {@link SchemaRegistryClient} and
 	 * {@link CacheManager}.
 	 * @param schemaRegistryClient the {@link SchemaRegistryClient} used to interact with
 	 * the schema registry server.
 	 * @param cacheManager instance of {@link CacheManager} to cache parsed schemas. If
 	 * caching is not required use {@link NoOpCacheManager}
+	 * @param manager instance of {@link AvroSchemaServiceManager} to manage schemas.
 	 */
 	public AvroSchemaRegistryClientMessageConverter(
-			SchemaRegistryClient schemaRegistryClient, CacheManager cacheManager) {
-		super(Collections.singletonList(DEFAULT_AVRO_MIME_TYPE));
+			SchemaRegistryClient schemaRegistryClient, CacheManager cacheManager, AvroSchemaServiceManager manager) {
```

Comment:
Public API changes here. We should retain the original constructor and delegate somehow to the new one internally.

Reply:
changed as suggested